sdcm/remote.py

```diff
@@ -43,6 +44,34 @@ class OutputCheckError(Exception):
     pass
 
 
+class CmdExecTimeoutExceeded(Exception):
+
+    """Exception which should be raised if
+    command execution time exceeds predefined timeout
+
+    """
+    pass
+
+
+class ConnectionCmdTimeout(Connection):
+
+    def __init__(self, host, user=None, port=None, config=None, gateway=None, forward_agent=None, connect_timeout=None, connect_kwargs=None):
```

Comment:
why no use `def __init__(self, *arg, **kwargs)` since you don't care about them, why not just pass anything ?

Reply:
The reason was, that invoke is checking what parameters were passed in init state of its object. So i want not allow to pass any additional parameters to constructor except allowed by Fabric.